app/src/main/java/com/example/notificationsapp/FirstFragment.kt

```diff
@@ -0,0 +1,88 @@
+package com.example.notificationsapp
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
+import android.os.Build
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.fragment.app.Fragment
+import com.example.notificationsapp.databinding.FragmentFirstBinding
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+
+const val IS_SHOW_NOTIFICATION_PREFERENCE_NAME = "isShowNotification"
```

Comment:
all actions with shared preferences should be incapsulated in the repository. and those keys as well